relay-event-schema/src/protocol/span.rs

```diff
@@ -277,6 +277,18 @@ pub struct SpanData {
     #[metastructure(field = "url.scheme")]
     pub url_scheme: Annotated<Value>,
 
+    /// Profile ID
+    #[metastructure(field = "profile_id")]
+    pub profile_id: Annotated<Value>,
```

Comment:
`profile_id` exists as a top-level field on the span, and it automatically gets picked up by kafka consumer. Should we use it?

Reply:
Hey Joris, good point, I dropped the changes to `profile_id` in span data.
Should be able to just pass `profile_id` at the top level, will update the sdks.